analyzers/src/SonarAnalyzer.Common/Facade/ISyntaxKindFacade.cs

```diff
@@ -43,10 +45,12 @@ public interface ISyntaxKindFacade<out TSyntaxKind>
     abstract TSyntaxKind ParameterList { get; }
     abstract TSyntaxKind ReturnStatement { get; }
     abstract TSyntaxKind SimpleAssignment { get; }
+    abstract TSyntaxKind SimpleCommentTrivia { get; }
     abstract TSyntaxKind SimpleMemberAccessExpression { get; }
     abstract TSyntaxKind[] StringLiteralExpressions { get; }
     abstract TSyntaxKind StructDeclaration { get; }
     abstract TSyntaxKind[] TypeDeclaration { get; }
+    abstract TSyntaxKind WhitespaceTrivia { get; }
```

Comment:
Can you move the LeftShiftExpression and the following properties to their right places?

Reply:
Moved them on the implementations as well

Reply:
They are not moved in analyzers/src/SonarAnalyzer.VisualBasic/Facade/VisualBasicSyntaxKindFacade.cs the yet.